docs/developer-guide/etendo-classic/how-to-guides/How_To_Create_a_Module.md

```diff
@@ -0,0 +1,209 @@
+---
+tags: 
+  - Etendo Classic
+  - Modularity
+  - Create a module
+  - Package a module
+  - Module
+---
+
+#  How To Create a Module
+
+##  Overview
+
+Etendo uses the concept of Modularity. Modularity provides
+developers with the means to be able to develop, publish and distribute their
+work in a structured and controlled manner. For users of Etendo Classic,
+Modularity makes it possible to download, install and update custom
+developments.
+
+This section describes how to create a new module. It is of
+particular interest as it describes the first steps which should be followed
+in all of the other sections in this developer's guide and in Etendo Classic
+custom development in general.
+
+##  Introduction to Modularity
+
+The objectives of modularity are:
+
+  * To make it easier to contribute to Etendo Classic by allowing distributed and decoupled development and maintenance of optional features. 
+  * To provide the community with a rich set of extensions to meet their unique business requirements without bloating the core product. 
+  * To shorten the implementation cycles by enabling system integrators to develop very specific functionality to meet the unique needs of a particular market. 
+
+A module is a piece of additional functionality that can be deployed
+optionally and independently on top of Etendo Classic. Examples of modules are:
+additional reports, additional windows, connectors, content packs
+(translations, chart of accounts, list of tax codes, product categories, etc).
+
+!!!info
+    For a detailed description on the Etendo Classic modularity concept, see the
+    [Modularity Guide](/concepts/Modularity_Concepts.md).
+
+This section explains only the standard module type, because that is most
+relevant for the other sections.
+
+All new development should be done as a part of a module. Only modules which
+are marked as _In Development_ can be used in the development process. A
+module can consist of the following software artifacts:
+
+  * Application dictionary components: all new application dictionary artifacts must be associated with a module. Application dictionary windows have a field that enables you to link an application dictionary component to a module. 
+  * Software resources: these are artifacts not defined in Etendo Classic itself but, for example, Java classes, libraries, web resources (images, etc.), configuration files, etc. To maintain the relationship with the module, software resources need to be located in the module's directory inside the Etendo Classic development project. Each module has its own unique directory. 
+  * Reference data: modules can be distributed with their own set of reference business data, for example, product categories, tax code, etc. The reference data is defined in datasets which can be related to a module. 
+
+The process of developing a module has three main steps:
+
+  1. Register your module in the Application Dictionary and in the central repository. 
```

Comment:
The concept of central repository no longer exists, customization modules are published by partners in their own repositories.